x-pack/plugins/security/server/lib/authentication/providers/__tests__/token.js

```diff
@@ -176,26 +176,6 @@ describe('TokenAuthenticationProvider', () => {
       expect(authenticationResult.notHandled()).to.be(true);
     });
 
-    it('fails if state contains invalid credentials.', async () => {
```

Comment:
__note:__ we had a similar one below, that I've repurposed to test `500` errors.